### PR TITLE
Blacklist all the things.

### PR DESCRIPTION
**Pull request by athoune**

### CODE REVIEW

**Review by nirae — APPROVED** [review]

**Comment by gawel** [comment]
it's useless if you do not support wildcard

**Comment by athoune** [comment]
See #5

**Comment by nirae** [comment]
Add wildcard support, using https://golang.org/pkg/path/filepath/#Match

File: blacklist/blacklist.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	}
+	return ok
+}
+
+func ReadBlacklistFile(path string) error {
+	raw, err := ioutil.ReadFile(path)
+	if err != nil {
+		return err
+	}
+	for _, line := range strings.Split(string(raw), "\n") {
```
Comment (athoune) [inline blacklist/blacklist.go]: It looks like https://golang.org/pkg/bytes/#Buffer.ReadString

**Review by athoune — COMMENTED** [review]
Readlines pattern.

**Pull request merged**